src/Cargo.lock

```diff
@@ -2110,7 +2110,6 @@ dependencies = [
 name = "rustc_codegen_utils"
 version = "0.0.0"
 dependencies = [
- "ar 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
No

Reply:
I was asking because there's a "checksum" line that would also be removed if this was the last use of `ar`.
I just checked and `rustc_driver` also depends on `ar`.

Reply:
what??? how did tidy pass then? i removed ar as whitelisted dep

Reply:
Removed unused dep